lib/fhir_client/client.rb

```diff
@@ -301,7 +301,7 @@ def scrubbed_response_headers(result)
 
     def get(path, headers)
       url = Addressable::URI.parse(build_url(path)).to_s
-      FHIR.logger.info "GETTING: #{url}"
+      FHIR.logger.debug "GETTING: #{url}"
```

Comment:
Personally: I think it might be nice to leave this one as `info`. Other than a search by MRN or something, it shouldn't be much of a risk. But I feel like it would be nice to not have to set debug level to know that a request was made, or to which resources.

Reply:
@wesrich do you have an alternative solution to his problem that you'd like to see him implement instead? I don't really care about the log level. @mtparet's alternative was to filter out secrets, which I am fine with.

Reply:
@wesrich I was thinking about changing the log level because I think most of rails related gems are not so verbose at this level. But I'm open to implement another solution.

Reply:
All that said ... I'm not entirely opposed to this change as is, it just leaves you not knowing what request was made (if any) without switching to debug mode. So if an error does happen or a user reports that they're seeing unexpected data, it may not be as obvious if the request was sent off properly (we sent the URL we expected, but something went wrong after), or if the request was bad to begin with.